client/src/components/ProductPreview.js

```diff
@@ -9,34 +9,38 @@ const Container = styled.div`
   justify-content: ${(props) => (props.primary ? "flex-end" : "center")};
   position: relative;
   width: 100%;
-  background-image: ${(props) => (props.primary ? "var(--gradient)" : "none")};
+  margin: ${(props) => (props.primary ? "0" : "5px")};
+  background-image: ${(props) => (props.primary ? "var(--gradient)" : "red")};
+
   > :last-child {
-    margin-bottom: ${(props) => (props.primary ? "30px" : "0")};
+    margin-bottom: ${(props) => (props.primary ? "15px" : "0")};
   }
   img {
     height: auto;
-    width: 100%;
-    border-radius: 50px;
-    padding: 10px;
+    width: ${(props) => (props.primary ? "100%" : "80%")};
   }
   h2 {
     color: var(--primary-color);
   }
 `;
 
-const LargeCard = ({ title, primary, src, alt, size }) => {
+const ProductPreview = ({ title, primary, src, alt, size }) => {
   return (
     <Container primary={primary}>
       {primary ? <h2>Our Pick of the Day</h2> : <></>}
       <Image src={src} alt={alt} size={size} />
-      <TextContainer title={title} primary={true} />
+      {size === "small" ? (
+        <></>
+      ) : (
+        <TextContainer title={title} primary={true} />
+      )}
```

Comment:
```suggestion
      {!size === "small" ? 
        <TextContainer title={title} primary={true} />
      }
```

Reply:
It did not work without the `:` 
I kept getting an error: `Unexpected token expected : : `